.github/workflows/docker-image.yml

```diff
@@ -511,6 +511,66 @@ jobs:
       - name: ensure httpd.service is running
         run:  sudo systemctl is-active --quiet httpd.service
 
+  systemd-user-enable-validate:
+    runs-on: ubuntu-latest
+    needs: [ build , build-systemd-amd ]
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: loginctl enable-linger
+        run: loginctl enable-linger $(whoami)
+
+      - name: Wait for enable-linger
+        run: sleep 3s
+
+      - name: XDG_RUNTIME_DIR
+        run: export XDG_RUNTIME_DIR=/run/user/1000
```

Comment:
I doubt this would make a difference but what about swapping 1000 for $UID or id -u

Reply:
I'm trying to leave in 1000 because it's hard-coded in the systemd.go - it shouldn't be though :)

Reply:
i stopped being lazy and updated that ^^